packages/app-project/src/screens/ClassifyPage/ClassifyPageContainer.js

```diff
@@ -15,14 +16,9 @@ function ClassifyPageContainer ({
   but can be reset by the Classifier component via onSubjectReset().
   This state does not change via components of the prioritized subjects UI (Next/Prev buttons) */
   const [selectedSubjectID, setSelectedSubjectID] = useState(subjectID)
+  const assignedWorkflowLevel = useAssignedLevel(assignedWorkflowID)
 
-  let assignedWorkflow
   let allowedWorkflows = workflows.slice()
-  let assignedWorkflowLevel = 1
-  if (assignedWorkflowID) {
-    assignedWorkflow = workflows.find(workflow => workflow.id === assignedWorkflowID)
```

Comment:
@shaunanoordin this was the line that caused the original bug. It assumes that all workflows, even completed ones, will be present in the `workflows` array.